src/runtime/plugin.js

```diff
@@ -0,0 +1,84 @@
+import Vue from 'vue'
+import { Strapi } from '@nuxtjs/strapi/runtime'
```

Comment:
This is the line which is causing the bug preventing the library to load properly: I tried replacing the import with `@nuxtjs/strapi/dist/runtime` but I still get the error.

Reply:
Should be fixed with the new alias `~strapi`, can you try again?